compiler/rustc_span/src/lib.rs

```diff
@@ -1036,6 +1036,15 @@ pub enum SourceFileHashAlgorithm {
     Sha1,
 }
 
+impl ToString for SourceFileHashAlgorithm {
```

Comment:
If you implement `Display` you don't need to manually call `.to_string()` when writing the dep file.

Reply:
Done